release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -56,6 +56,17 @@ This release adds improvements related to the following components and concepts.
 
 User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
 
+[id="ocp-4-14-admin-ack-upgrading"]
+==== Required administrator acknowledgment when upgrading from {product-title} 4.13 to 4.14
+
+{product-title} 4.14 uses Kubernetes 1.27, which removed a xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-removed-kube-1-27-apis[deprecated API].
+
+A cluster administrator must provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.13 to 4.14. This is to help prevent issues after upgrading to {product-title} 4.14, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this is done, the administrator can provide the administrator acknowledgment.
```

Comment:
Should we stick with upgrade or update across the entry for consistency? I do understand that both are correctly used. I'm asking more for my own learning.

Reply:
Good question! We do prefer "update" (per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update), so I will do a sweep to update the instances I can. Thanks!